ios/fpjs_pro_plugin.podspec.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "fpjs_pro_plugin",
+  "version": "3.3.2",
+  "summary": "Flutter plugin for FingerprintJS Pro.",
+  "description": "Flutter plugin for FingerprintJS Pro.",
+  "homepage": "https://github.com/fingerprintjs/fingerprintjs-pro-flutter",
+  "license": {
+    "file": "../LICENSE"
+  },
+  "authors": {
+    "FingerprintJS, Inc (https://fingerprint.com)": "boris.lobanov@fingerprintjs.com"
```

Comment:
why is this line different than [the RN one](https://github.com/fingerprintjs/fingerprintjs-pro-react-native/pull/147/files#diff-15436650b78a9c696dc3e91ecaf8bf71e9d5e0e949db5035d81864e66f6df157R12)?

Reply:
@erayaydin let's fix it in this PR